dnsfilter/README.md

```diff
@@ -1,43 +1,54 @@
-# Agent Check: DNSFilter
+# DNSFilter
 
 ## Overview
 
-This check monitors [DNSFilter][1].
+[DNSFilter][1] is a cloud-based content filtering and threat protection by blocking internet threats at the DNS layer. It protects organizations by preventing access to malicious domains, phishing sites, and other cyber threats, ensuring a safer internet experience.
```

Comment:
```suggestion
[DNSFilter][1] is a cloud-based content filtering tool that blocks internet threats at the DNS layer. It protects organizations by preventing access to malicious domains, phishing sites, and other cyber threats, ensuring a safer internet experience.
```
The first sentence is not grammatically correct. I made one possible edit, but feel free to revise to your preferences.